ArchivedPage.php

```diff
@@ -87,24 +127,122 @@
+        INNER JOIN taskList tl
+        ON t.`tl.listID` = tl.listID WHERE `taskID`=?');
+                    $stmt->execute(array($task['taskID']));       
+                    $task = $stmt->fetch();
+                    //Now we have our task info in the $task variable, lets output the information for the user.
+                    ?>
+            <form action="listView.php?board=<?php echo $board['boardID']; ?>" class="form-newList" method="POST">
+            <tr>
+              <th>ID</th>
+              <td><?php echo $task['taskID'];?></td>
+              <?php if ($edit) { ?><td>Modify Values and Submit Below</td><?php } ?>
+            </tr>
+            <tr>
+              <th scope="row">Title</th>
+              <td><?php echo $task['taskTitle'];?></td>
+              <?php if ($edit) { ?><td><input type="text" class="form-control" name="taskTitleUpdate" value="<?php echo $task['taskTitle']; ?>"></td><?php } ?>
+            </tr>
+            <tr>
+              <th scope="row">Description</th>
+              <td><?php echo $task['taskDescription'];?></td>
+              <?php if ($edit) { ?><td><input type="text" class="form-control" name="descriptionUpdate" value="<?php echo $task['taskDescription']; ?>"></td><?php } ?>
+            </tr>
+            <tr>
+              <th scope="row">Due Date</th>
+              <td><?php echo date("Y-m-d H:i:s", $task['dueDate']);?></td>
+              <?php if ($edit) { ?><td><input class="form-control" type="datetime-local" value="unchanged" name="updateDueDate" id="updateDueDate"></td><?php } ?>
+            
+            </tr>
+            <tr>
+              <th scope="row">Date Created</th>
+              <td><?php echo date("Y-m-d H:i:s", $task['taskDateCreated']);?></td>
+              <?php if ($edit) { ?><td>Cannot Modify Date Created</td><?php } ?>
+            </tr>
+            <tr>
+              <th scope="row">Importance</th>
+              <td><?php echo $task['importance'];?></td>
+              <?php if ($edit) { ?><td><input type="text" class="form-control" name="importanceUpdate" value="<?php echo $task['importance']; ?>"></td><?php } ?>
+            </tr>
+            <tr>
+              <th scope="row">Type Of Work</th>
+              <td><?php echo $task['typeOfWork'];?></td>
+              <?php if ($edit) { ?><td><input type="text" class="form-control" name="typeOfWorkUpdate" value="<?php echo $task['typeOfWork']; ?>"></td><?php } ?>
+            </tr>
+            <tr>
+              <th scope="row">Course</th>
+              <td><?php echo $task['courseTitle'];?></td>
+              <?php if ($edit) { ?><td><select name = "updateCourseID">
+                    <option value="<?php echo $task['courseID']; ?>"><?php echo $task['courseTitle']; ?></option>
+                        <?php                    
+                            $query = "SELECT courseID,courseTitle FROM course";
+                            $courseTitles = mysqli_query($conn, $query);
```

Comment:
To maintain consistency, please use PDO to execute query calls - this file has already made use of it(lines 134 to 147).